File: finished/palintrome.cpp
#include <iostream>
using namespace std;

int main(){
    int num;
    cout << "Enter a number : ";
    cin >> num;

    int original = num;
    int rev = 0;
    while (num != 0)
    {
        rev = rev*10 + num%10;
        num /= 10;
    }
    if (rev==original)
    {
        cout << "Number is a Palintrome";
    }else
    {
        cout << "Number is not a Palintrome";
    }
}

File: finished/palintrome.py
num = int(input("Enter number for Palintrome test : "))
original = num
rev = 0

while int(num) != 0:
    rev = rev*10 + int(num)%10
    num = num / 10
if rev == original:
    print(original,"Is a Palintrome")
else:
    print(original,"Is NOT a Palintrome")